gradle-plugin/src/main/kotlin/com/google/devtools/ksp/gradle/KspAATask.kt

```diff
@@ -305,6 +320,12 @@ abstract class KspGradleConfig @Inject constructor() {
     @get:PathSensitive(PathSensitivity.RELATIVE)
     abstract val javaSourceRoots: ConfigurableFileCollection
 
+    @get:InputFiles
+    @get:SkipWhenEmpty
+    @get:IgnoreEmptyDirectories
+    @get:PathSensitive(PathSensitivity.RELATIVE)
+    abstract val resourceRoots: ListProperty<Directory>
```

Comment:
I did not make resource handling incremental. Not sure if it's needed and how the API would look. Maybe there would be a separate `getNewResources` or something like that.